controllers/storagecluster/storagecluster_controller.go

```diff
@@ -144,14 +150,63 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	)
 
-	builder := ctrl.NewControllerManagedBy(mgr).
+	noobaaIgnoreTimeUpdatePredicate := predicate.Funcs{
+		UpdateFunc: func(e event.UpdateEvent) bool {
+			if e.ObjectOld == nil || e.ObjectNew == nil {
+				return false
+			}
+			oldObj := e.ObjectOld.(*nbv1.NooBaa)
+			newObj := e.ObjectNew.(*nbv1.NooBaa)
+
+			diff := cmp.Diff(
+				oldObj, newObj,
+				cmpopts.IgnoreFields(metav1.ObjectMeta{}, "ManagedFields", "ResourceVersion"),
+				cmpopts.IgnoreFields(conditionsv1.Condition{}, "LastHeartbeatTime", "LastTransitionTime"),
+			)
+
+			return diff != ""
+		},
+	}
+
+	cephClusterIgnoreTimeUpdatePredicate := predicate.Funcs{
+		UpdateFunc: func(e event.UpdateEvent) bool {
+			if e.ObjectOld == nil || e.ObjectNew == nil {
+				return false
+			}
+			oldObj := e.ObjectOld.(*cephv1.CephCluster)
+			newObj := e.ObjectNew.(*cephv1.CephCluster)
+
+			diff := cmp.Diff(
```

Comment:
Why does OCS operator need to reconcile if anything changed in the CephCluster status? Normally I would expect reconcile is only needed when the `spec` is updated. Otherwise we're going to be chasing exceptions in this list every time another cephcluster status is updated too frequently.

Reply:
And did something change in OCS operator recently about this? Or the reconciles weren't noticed as much previously?

Reply:
We are updating the storage cluster status based on the ceph cluster status, that's why we are watching for the status.

Reply:
Things are not changed in the ocs. We got an customer case where ocs-operator is consuming around 107 gb of space for logs in there loki bucket.

Reply:
got it thanks